src/pages/_document.js

```diff
@@ -3,10 +3,28 @@ import Document, { Head, Main, NextScript } from 'next/document'
 
 export default class DefaultDocument extends Document {
   render() {
+    const FB_SCRIPT = `
+      window.fbAsyncInit = function() {
+        FB.init({
+          appId      : '240115020081725',
```

Comment:
Same here, this appID should be a env var so it's changed in the app configurations.